src/test/java/com/google/cloud/solutions/autotokenize/pipeline/DlpInspectionPipelineIT.java

```diff
@@ -93,7 +92,7 @@
   private final Clock fixedClock;
 
   private transient String outputFolder;
-  public static transient DB testDBInstance;
+  private static JdbcDatabaseContainer<?> databaseContainer;
```

Comment:
remove static and use `@After` instead of `@AfterClass`

Reply:
Fixed